scripts/scripts-1.js

```diff
@@ -0,0 +1,69 @@
+const userName = prompt("Яке Ваше імʼя?");
+
+const userSurname = prompt("Яке Ваше прізвище?");
+
+const userFavoriteNumber = +prompt("Яке Ваше улюблене число?");
+
+alert(`Вітаю, ${userName} ${userSurname}! Ваше улюблене число - ${userFavoriteNumber}`);
+
+console.log(`Вітаю, ${userName} ${userSurname}! Ваше улюблене число - ${userFavoriteNumber}`);
+
+const firstClientNumber = +prompt("Введіть перше число");
+
+const secondClientNumber = +prompt("Введіть друге число");
+
+const addition = (firstClientNumber + secondClientNumber);
+
+const subtraction = (firstClientNumber - secondClientNumber);
+
+const multiplication = (firstClientNumber * secondClientNumber);
+
+const division = (firstClientNumber / secondClientNumber);
+
+alert(`Сума - ${addition}, різниця - ${subtraction}, добуток - ${multiplication}, ділення - ${division}`);
+
+console.log(`Сума - ${addition}, різниця - ${subtraction}, добуток - ${multiplication}, ділення - ${division}`);
+
+const userAgeQuestion = +prompt("В якому році Ви народились");
+
+let howOldIsTheUser;
+
+const dataNow = 2022;
+
+const userAgeResult = (2022 - userAgeQuestion);
```

Comment:
```suggestion
const userAgeResult = (dataNow - userAgeQuestion);
```
If you already have a const for the date it should be used here, to avoid "magic" values in the code